_assignments/mp1.md

```diff
@@ -12,35 +12,35 @@ rubric:
   -
     name: The Room
     points: 10
-    description: Complete and enclosed room
+    description: Complete and enclosed room.

```

Comment:
[nit] If we're going to add periods here, we should also make these into complete sentences. This applies to a few others here.